docs/examples/opentelemetry-example-app/setup.cfg

```diff
@@ -41,11 +41,11 @@ packages = find_namespace:
 zip_safe = False
 include_package_data = True
 install_requires = 
-    opentelemetry-api == 0.16.dev0
-    opentelemetry-sdk == 0.16.dev0
-    opentelemetry-instrumentation-requests == 0.15.b0
-    opentelemetry-instrumentation-flask == 0.15.b0
-    opentelemetry-instrumentation-grpc == 0.15.b0
+    opentelemetry-api == 0.16b0
+    opentelemetry-sdk == 0.16b0
+    opentelemetry-instrumentation-requests == 0.16b0
+    opentelemetry-instrumentation-flask == 0.16b0
+    opentelemetry-instrumentation-grpc == 0.16b0
```

Comment:
I saw that tests are failing because it doesn't find this `grpc` package at `0.16b0` but I'm not sure why the other instrumentation packages don't fail...

Reply:
It looks like you have the correct `CONTRIB_REPO_SHA` because that SHA has `grpc` @ `0.16b0` https://github.com/open-telemetry/opentelemetry-python-contrib/pull/204/files#diff-448a4e5ed17ab6b9fc83852e5beef5bf39d8e1d09e39fe9561be0de075544201R15